server/src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -1210,19 +1211,20 @@ protected void updateLocalTransforms() {
 	 * @param ankle the second Quaternion
 	 * @return the rotated Quaternion
 	 */
-	Quaternion extendedKneeYawRoll(Quaternion knee, Quaternion ankle) {
+	private Quaternion extendedKneeYawRoll(Quaternion knee, Quaternion ankle) {
+		knee = knee.clone();
```

Comment:
I myself was confused why this is being cloned, based off that, I'd like to have a comment so nobody removes it without understanding why it's done
```suggestion
		// Clone the knee as we're returning a separate quaternion
		knee = knee.clone();
```